openprescribing/outliers/tests.py

```diff
@@ -0,0 +1,35 @@
+from django.test import TestCase
+from frontend.models import PCT
+from matrixstore.tests.data_factory import DataFactory
+from matrixstore.tests.matrixstore_factory import (
+    matrixstore_from_data_factory,
+    patch_global_matrixstore,
+)
+from outliers.build import prescribing_for_orgs
+
+
+class BuildTest(TestCase):
+    @classmethod
+    def setUpTestData(cls):
+        factory = DataFactory()
+        factory.create_all(
+            start_date="2018-06-01", num_months=6, num_practices=6, num_presentations=6
+        )
+        ccg = PCT.objects.create(code="ABC", org_type="CCG")
+        for i in range(6):
+            ccg.practice_set.create(code=f"ABC00{i}", setting=4)
+
+        cls._remove_patch = patch_global_matrixstore(
+            matrixstore_from_data_factory(factory)
+        )
+
+    def test_build_smoke_test(self):
+        # The data produced by the DataFactory is not rich enough to test anything
+        # interesting, but this is a reasonable smoke test that the calculations work.
```

Comment:
Great, yes, this was all I thought it was worth doing. Glad to see that there was enough matrixstore factory stuff lying around (which I'd completely forgotten about) to make this relatively easy.